source/material_model/equation_of_state/thermodynamic_table_lookup.cc

```diff
@@ -351,6 +424,9 @@ namespace aspect
         // fill phase volume outputs if they exist
         if (NamedAdditionalMaterialOutputs<dim> *phase_volume_fractions_out = out.template get_additional_output<NamedAdditionalMaterialOutputs<dim> >())
           fill_phase_volume_fractions(in, volume_fractions, phase_volume_fractions_out);
+
+        if (PhaseOutputs<dim> *dominant_phases_out = out.template get_additional_output<PhaseOutputs<dim> >())
+          fill_dominant_phases(in, volume_fractions, dominant_phases_out);
```

Comment:
Keep the line above, but hand over a reference instead of a pointer like follows:
```suggestion
          fill_dominant_phases(in, volume_fractions, *dominant_phases_out);
```